tasks/prelim.yml

```diff
@@ -56,6 +56,19 @@
   tags:
       - always
 
+- name: ensure cronie is available
+  yum:
+      name: cronie
+  when:
+      - rhel_07_020030 or
+        rhel_07_020040 or
+        false
```

Comment:
What is the purpose of the `or false`?

Reply:
Allows reordering the above items without worrying about which one doesn't have an or at the end, and makes it so adding new items doesn't require adding or to an existing item

Reply:
But in this case, the list is quite short and unlikely to change frequently. I'm happy to remove it if that is preferred.

Reply:
I'm fine leaving it in I just wasn't sure what the purpose was lol. Now it makes sense.